packages/stainless/src/stl.ts

```diff
@@ -88,14 +89,14 @@ export type Endpoint<
   stl: Stl<UserContext, any>;
   endpoint: MethodAndUrl;
   response: Response;
-  config: EndpointConfig;
+  config: Config;
   path: Path;
   query: Query;
   body: Body;
   handler: Handler<
     UserContext &
       StlContext<
-        Endpoint<UserContext, MethodAndUrl, Path, Query, Body, Response>
+        Endpoint<UserContext, Config, MethodAndUrl, Path, Query, Body, Response>
```

Comment:
Our types are getting large. Might be worth considering whether the added convenience of magical context is worth the added complexity of implementation.

Reply:
Let's spend some time IRL on this tomorrow, curious to explore together with @jedwards1211

Reply:
You know looking at this now I see a way to avoid this recursion in the endpoint type

Reply:
Hopefully that was part of what caused some of the grief with .from!